docker-compose.yml

```diff
@@ -15,25 +15,27 @@ services:
       start_period: 30s
 
   minio:
-    image: minio/minio:RELEASE.2023-01-31T02-24-19Z
+    image: bitnami/minio:2024.4.6
+    environment:
+      MINIO_ROOT_USER: minioadmin
+      MINIO_ROOT_PASSWORD: minioadmin
     ports:
       - 9000:9000
       - 9001:9001
     volumes:
-      - minioVolume:/data
-    command: server /data --console-address ":9001"
+      - minioVolume:/bitnami/minio
 
   clamd:
     image: clamav/clamav:1.3
 
   mailcrab:
-    image: marlonb/mailcrab:v0.10.0
+    image: marlonb/mailcrab:v1.2.0
     ports:
       - 1080:1080
       - 1025:1025
 
   nginx:
-    image: nginx:1.21.6-alpine
```

Comment:
Is it worth ensuring this is consistent with our other environments for debugging purposes?